timer/timer.go

```diff
@@ -136,7 +137,10 @@ func (timer *Timer) newCron(t fission.TimeTrigger) *cron.Cron {
 		headers := map[string]string{
 			"X-Fission-Timer-Name": t.Name,
 		}
-		(*timer.publisher).Publish("", headers, fission.UrlForFunction(&t.Function))
+		respChan := make(chan string)
+		(*timer.publisher).Publish("", headers, fission.UrlForFunction(&t.Function), respChan)
+		fmt.Printf("Trigger %s get response from function: %s\n", t.Metadata.Name, <-respChan)
```

Comment:
This will block forever if there's no response. Also, it's better to be explicit and put the channel read on a separate line from the logging.

Also please use log.Printf not fmt.Printf